style/style.css

```diff
@@ -515,3 +516,118 @@ button:active {
   position: relative;
   top: -26px;
 }
+
+.contact {
+  background: hsl(234, 100%, 69%);
+  border-top-left-radius: 100px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 375px;
+  height: 880px;
+  position: relative;
+  left: -8px;
+  top: -12px;
+}
```

Comment:
Kindly make sure that your project sticks as much as possible to the Figma template provided. There are margins on the left and right sides that there are not in the Figma template. Please remove them. :pray: :pray: 

Your design
![image](https://user-images.githubusercontent.com/98363075/199248451-81e40d74-20fe-46bd-8d0b-e291ca36486e.png)

Figma design
![image](https://user-images.githubusercontent.com/98363075/199248877-14236b0b-480f-4231-861e-63bb753b71db.png)